recipes/TheanoLM/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "TheanoLM" %}
+{% set version = "1.2.0" %}
+{% set sha256 = "ebb312c34ad0bfea950f412f4657fe9ae6dc8ed8e904bb199ceef03faf50ff80" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [py<34]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - theano
+    - h5py
+
+
+test:
+  imports:
+    - theanolm
+    - theanolm.commands
+    - theanolm.network
+    - theanolm.parsing
+    - theanolm.scoring
+    - theanolm.training
+    - theanolm.vocabulary
+  commands:
+    - theanolm --help
+
+
+about:
+  home: https://github.com/senarvi/theanolm
+  license: Apache 2.0 
+  license_file: LICENSE.txt
+  summary: 'Recurrent neural network language modeling tool implemented using Theano'
+
+  description: |
+    Recurrent neural network language modeling tool implemented using Theano
+  doc_url: http://theanolm.readthedocs.org/
+  dev_url: https://github.com/senarvi/theanolm
+
+extra:
+  recipe-maintainers:
+    - psmit
+    - senarvi
```

Comment:
@senarvi, are you happy to [be a maintainer](http://conda-forge.readthedocs.io/en/latest/recipe.html#maintainer-role) for the conda-forge package of TheanoLM?

Reply:
You can add me as a maintainer. This is the first time I'm looking at such a recipe but I'll talk with @psmit about things that I should know.